dev/bin/interaction/environment-example

```diff
@@ -0,0 +1,11 @@
+# This is an example local environment file.
+# It can be used to configure required variables for your local environment
+# without the need of exporting them every time, like offline tokens etc.
+# If you want to use it, make a copy of this named "environment" in this same folder.
+
+MINIKUBE_IP=$(minikube ip)
+KEYCLOAK_URL=http://$MINIKUBE_IP:30007
+MANAGER_URL=http://localhost:8080
```

Comment:
We should harmonize that. 
Those are defined in `dev/bin/credentials/local_env` and updated by `dev/bin/manager-run.sh` or `kustomize/startMinikubeDeployLocalDev.sh`

Reply:
but this is an unrelated file, isolated to these scripts

Reply:
The point then is that it would depend if you start the local env with `manager-run` or with manager in minikube with `startMinikubeDeployLocalDev`. Then the `MANAGER_URL` is different... but can still be modified manually.

I guess we need some rework on all our script to have some common base. 
I can take care of it after the merge if you want.